src/changelog/unreleased.md

```diff
@@ -52,6 +52,14 @@ changelog entry.
   to send specific data to be processed on the main thread.
 - Changed `EventLoopProxy::send_event` to `EventLoopProxy::wake_up`, it now
   only wakes up the loop.
+- `ApplicationHandler::create|destroy_surfaces()` was split off from
+  `ApplicationHandler::resumed/suspended()`.
+
+  `ApplicationHandler::create_surfaces()` should, for portability reasons to
+  Android, be the only place to create render surfaces.
```

Comment:
What do you mean exactly?
Couple of lines below it the new behavior of `resumed/suspended()` is explained.

Reply:
I'd probably write it more clear that the `resume` logic should be moved to the new function and that `resumed` serves now completely different purpose, because before we advertised it as a place to create a surfaces.

Alternative is to rename resumed to something else, so users don't have to read into the changelog a lot about why their app is not working anymore.

Reply:
> I'd probably write it more clear that the `resume` logic should be moved to the new function and that `resumed` serves now completely different purpose, because before we advertised it as a place to create a surfaces.

That's not really correct though, the current `resume()` functionality isn't new, its just split off. So users can't just move their `resume()` logic to `can_create_surfaces()`, they have to actually just split it off, like it says in the changelog already.

I'm happy to review a suggestion on what you have in mind though.

> Alternative is to rename resumed to something else, so users don't have to read into the changelog a lot about why their app is not working anymore.

I don't think we have to worry about that because it will fail to build anyway unless `can_create_surfaces()` is implemented. Hopefully at that point users should be looking at the documentation to implement it.

Reply:
hm, that's a good point, probably fine then.